ngx_http_uploadprogress_module.c

```diff
@@ -640,17 +645,13 @@ ngx_http_reportuploads_handler(ngx_http_request_t * r)
         expires->next = NULL;
 
         expires->hash = 1;
-        expires->key.len = sizeof("Expires") - 1;
-        expires->key.data = (u_char *) "Expires";
+        expires->key = (ngx_str_t)ngx_string("Expires");
     }
 
-    len = sizeof("Mon, 28 Sep 1970 06:00:00 GMT");
-    expires->value.len = len - 1;
+    expires->value = (ngx_str_t)ngx_string("Mon, 28 Sep 1970 06:00:00 GMT");
```

Comment:
I think we should use `Thu, 01 Jan 1970 00:00:01 GMT` here which is what the value was set too below. I don't explain nor remember why I did chose Sep 28 over than Jan 01 for the size (and not the value).

Reply:
Updated.